serde_codegen_internals/src/attr.rs

```diff
@@ -270,6 +278,17 @@ impl Item {
                         }
                     }
 
+                    // Parse `#[serde(types(from = "type", into = "type", as = "type"))]
+                    MetaItem(List(ref name, ref meta_items)) if name == "types" => {
```

Comment:
I would prefer not to have this `types(...)` layer. I think `#[serde(into = "...")]` is clear enough. What were the reasons for including it?